languageFactories/language-factory-core/src/main/scala/cromwell/languages/config/LanguageConfiguration.scala

```diff
@@ -21,9 +21,10 @@ object LanguageConfiguration {
     val languages = LanguageNames.toList map { languageName =>
 
       val languageConfig = LanguagesConfig.getConfig(languageName)
-      val defaultVersionName: Option[String] = if (LanguagesConfig.hasPath("default")) { Option(LanguagesConfig.getString("default")) } else None
       val versionSet = languageConfig.getConfig("versions")
-      val languageVersionNames: Set[String] = versionSet.entrySet().asScala.map(findFirstKey).filterNot(_ == "default").toSet
+      val allLanguageVersionNames: Set[String] = versionSet.entrySet().asScala.map(findFirstKey).toSet
```

Comment:
This is a bug in the baseline code. The expression that was used for `defaultVersionName` is essentially the same as the expression used for `DefaultLanguageName` on line 16. This means that the default language works out to be "WDL" and the default version of "WDL" is "WDL". Somehow on 2.12 this junk result still ended up choosing draft 2 as the default language factory, but on 2.13 this chose draft 3. When WDLs without an explicit `version development` were submitted without explicitly specifying an language version, the draft 3 language factory rejected them on their first token.

Reply:
Yeah this was kind of a 😬 finding